client/public/clientJS/stripe.js

```diff
@@ -123,6 +123,26 @@ let addCashDonation = function (originalAmount, amount) {
   userDonation.innerHTML = '$' + Math.floor(amount* 100) / 100;
 }
 
+let showCustomAmountInput = () => {
+  let inputAmountElement = document.getElementById("customAmountValue");
+  const isVisible = inputAmountElement.style.visibility === 'hidden';
+  // if users hides the input reset total amount to 0
+  if (isVisible === false) {
```

Comment:
if !isVisible?

Reply:
i prefer explicit check in cases when comparing to negatives

Reply:
why, whats the difference between your code and !?